source/tests/passing.d

```diff
@@ -4737,3 +4737,21 @@ immutable test221 = q{--- test221
 	}
 	void debugCallback() {}
 };
+
+
+@TestInfo()
+immutable test222 = q{--- test222
+	@extern(module, "kernel32")
+	i32 test(i32 number) {
+		i32 result;
```

Comment:
This should not need a body. But I should note to myself that directly applying `@extern` to functions with body should be an error.

Reply:
Oh, that's a good test case scenario

Reply:
There is a file for tests that should fail

Reply:
Got it